specs/001-deploy-time-providers-l1/plan.md

```
@@ -0,0 +1,910 @@
+		finalConfig = &RunYamlConfig{
+			Version:   2,
+			Providers: make(map[string][]ProviderConfigEntry),
+		}
+	}
+
+	// Validate no duplicate external provider IDs
+	if err := validateNoDuplicateExternalProviderIDs(instance); err != nil {
+		return nil, nil, err
+	}
+
+	// Merge external providers
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)